docs/components/connectors/protocol/http-webhook.md

```diff
@@ -157,6 +157,12 @@ You can still use HTTP Webhook connector in Desktop Modeler, or with your Camund
 In that case, HTTP Webhook connector deployments and URLs will not be displayed in Modeler.
 :::
 
+For self-managed installations using the desktop modeler, the webhook URL format is:
+
+`http(s)://<base URL>/<contextPath>/inbound/<webhook ID>`
```

Comment:
@mathias-vandaele This is AFAIK not related to the Desktop- (or Web-) Modeler. 


@mesellings 

Not sure how to best link to the specific configs like [Docker](https://docs.camunda.io/docs/next/self-managed/installation-methods/docker/#connectors) and this [general page](https://docs.camunda.io/docs/next/self-managed/components/connectors/connectors-configuration) for example as well as https://docs.camunda.io/docs/next/self-managed/installation-methods/helm/#install-connectors

We also have tbis one which contains the actual context path mentioned here: https://artifacthub.io/packages/helm/camunda/camunda-platform#connectors-parameters